CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## (in progress)
 
+## [2.0.1](https://github.com/folio-org/ui-quick-marc/tree/v2.0.1) (2020-11-11)
+[Full Changelog](https://github.com/folio-org/ui-quick-marc/compare/v1.1.0...v2.0.1)
```

Comment:
does it make sense to put `v2.0.0...v2.0.1`?

Reply:
fixed